packages/roosterjs-react-emoji/lib/components/emoji.scss

```diff
@@ -12,8 +12,56 @@
     div {
         font-size: 18px;
     }
+
+    button {
+        width: $buttonWidth;
+    }
+}
+
+.emojiTextInput {
+    padding: 6px;
+}
+
+.fullList {
+    position: relative;
 }
 
-.header {
-    border-bottom: solid 1px #aaa;
-}
\ No newline at end of file
+.pivot {
+    $buttonColor: #f4f4f4;
+    background-color: $buttonColor;
+    height: $buttonHeight;
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    border-bottom: solid 1px #dadada;
+    padding: 0 $paneHorizontalPadding;
+
+    button {
+        background: $buttonColor;
+        height: $buttonHeight;
+        width: $buttonWidth;
+        margin-right: 0 !important;
+
+        span {
+            flex-grow: 1;
+        }
+    }
+}
+
+.partialList,
+.fullListBody {
+    max-height: 245px;
+    overflow: hidden;
+    overflow-y: auto;
+}
+
+.fullListContentContainer {
+    margin-top: $buttonHeight;
+}
+
+.partialListContent,
+.fullListContent {
+    width: $paneWidth;
```

Comment:
Do you plan to fix the IE11 issue in a separate PR?

Reply:
same pr